prog geometry/prog geometry/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace prog_geometry
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double x;
+            Console.WriteLine("Введите основание x:");
+            x = double.Parse(Console.ReadLine());
+            
+            
+
+            double y;
+            Console.WriteLine("Введите основание y:");
+            y = double.Parse(Console.ReadLine());
+
+
+            double z;
+            Console.WriteLine("Введите высоту z:");
+            z = double.Parse(Console.ReadLine());
+
+
+            double S;
```

Comment:
В C# локальные переменные принято назвать с маленькой буквы.
```suggestion
            double s;
```